### ISSUE TITLE
TX's sitting in the pool should display as locked balance 

### ISSUE THREAD

**Issue by Doy-lee** [loki-project/loki#398]
This is most likely a regression. The wallet is aware that there are transactions sitting in the tx pool for itself and notifies as such
```
Password needed (output received) - use the refresh command
```

Upon refresh, locked balance is not updated and the wallet on idle will continually ask for a refresh until it is included in a block.

Steps To Reproduce (On a local testnet)
1. Launch two wallets and a daemon
2. On wallet A, mine a couple of blocks
3. On wallet A, transfer to wallet B
4. Observe wallet B while it is idle, and ```Password needed (output received) - use the refresh command``` will appear
5. Refresh on wallet B, observe that locked balance remains at 0

**Comment by Doy-lee** [comment]
Interesting, I was under the impression it used to. The bug here I suppose is requesting a refresh unnecessarily whilst the TX is to be included in a block.

This issue and associated PR are likely related.

https://github.com/monero-project/monero/issues/5066
https://github.com/monero-project/monero/pull/5069

**Comment by Doy-lee** [comment]
Closed as duplicate with #644
